Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:20.11.1
```

Comment:
Please move Dockerfiles into subdirs based on apps. The compose file is a good idea!